.github/workflows/test.yml

```
@@ -0,0 +1,35 @@
+name: Test
+
+on: push
+
+jobs:
+  test:
+    name: Bootstrap app, run linters and tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [14, 16]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup node and restore cached dependencies
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+          cache: 'npm'
+
+      - name: Bootstrap React App
+        run: npx create-react-app test-app --template file:./
+
+      - name: Run test
+        working-directory: ./test-app
+        run: npm run lint
+      
+      - name: Run unit tests
+        working-directory: ./test-app
+        run: npm test
```

Should we change this to make it consistent?
```suggestion
        run: npm run test
```